src/js/models/Search.js

```diff
@@ -0,0 +1,44 @@
+import { errorMessage } from '../views/searchView';
+
+export default class Search {
+  constructor(query) {
+    this.query = query;
+    this.key = 'b4ea110422367ae01f9de8c612462846';
+    this.url = 'http://api.openweathermap.org/data/2.5/forecast?q=';
```

Comment:
Let's use `https` here to avoid the following error on a live link:

> Mixed Content: The page at '<URL>' was loaded over HTTPS, but requested an insecure resource '<URL>'. This request has been blocked; the content must be served over HTTPS.

See https://rawcdn.githack.com/vieiramanda11/js-weather-app/ea83931db6c50a2237635b6093f8a85bf5f252c3/dist/index.html